backend/app/services/llm/jobs.py

```
@@ -1183,22 +1269,26 @@ def execute_chain_job(
         try:
             with Session(engine) as session:
-                chain_record = create_llm_chain(
-                    session,
-                    job_id=job_uuid,
-                    project_id=project_id,
-                    organization_id=organization_id,
-                    total_blocks=len(request.blocks),
-                    input=serialize_input(request.query.input),
-                    configs=[block.model_dump(mode="json") for block in request.blocks],
-                )
+                chain_record = get_llm_chain_by_job_id(session, job_uuid)
```

Ruff (0.15.12)

[error] Undefined name `get_llm_chain_by_job_id`

(F821)